src/ledger-db/README.md

```diff
@@ -10,18 +10,16 @@ Implemented using Postgres.
 
 ### Environment Variables
 
-- `USE_DEFAULT_DATA`
-  - adds default data to ledger when initialized if `True`
-- `LOCAL_ROUTING_NUM`
-  - the routing number for our bank
-- `DEFAULT_ACCOUNT`
-  - the default user account to generate transactions for if `USE_DEFAULT_DATA` is `True`
-- `DEFAULT_DEPOSIT_ACCOUNT`
-  - the account number for the external bank account initiating deposits in default data
-- `DEFAULT_DEPOSIT_ROUTING`
-  - the routing number for the external bank account initiating deposits in default data
-
-- ConfigMap `postgres-config`:
+- ConfigMap `environment-config`:
+  - `LOCAL_ROUTING_NUM`
+    - the routing number for our bank
+
+- ConfigMap `default-data-config`:
+  - `USE_DEFAULT_DATA`
```

Comment:
Doesn't this still need the DEFAULT_ACCOUNT, DEFAULT_DEPOSIT_ACCOUNT, and DEFAULT_DEPOSIT_ROUTING?

Reply:
Yes it still needs those.  And it will need even more after #74 .  And possibly more in the future.  Instead of exhaustively including a line for every piece of default data, I figure it will be much easier to just include a reference the default data configuration.

Reply:
I'd argue if there are going to be that many, we might want to just keep them as constant variables in the script itself. This kind of data isn't really suited for environment variables anyway, and AFAIK these won't be used by multiple services

Reply:
> This kind of data isn't really suited for environment variables anyway

Very much agreed.  My next PR will be about consolidating the test data into some format that doesn't involve environment variables.

> AFAIK these won't be used by multiple services

As it is configured right now, both 'ledger-db' and 'accounts-db' are using these default test data variables.

Reply:
It is a different set of variables in those services though, right? (Except for the default user info, which I think does make sense as a global environment variable)

But yeah I agree, all that stuff can come in a different PR

Reply:
Okay, another PR it is.